lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -469,6 +469,11 @@ data WalletPutPassphraseData = WalletPutPassphraseData
     , newPassphrase :: !(ApiT (Passphrase "raw"))
     } deriving (Eq, Generic, Show)
 
+data ByronWalletPutPassphraseData = ByronWalletPutPassphraseData
+    { oldPassphrase :: !(Maybe (ApiT (Passphrase "raw")))
```

Comment:
This won't work unfortunately ... not only does the old passphrase can be empty, but it can also be less than 10 characters (which the new backend doesn't have as a limitation). 

My suggestion would be to have a `ApiT (Passphrase "byron-raw")`, for which we can define a different minimum length that is different from `Passphrase "raw"`, and coerce the types in the API handlers.

Reply:
done